app/(protected)/settings/page.tsx

```diff
@@ -1,21 +1,218 @@
-import { auth, signOut } from "@/auth";
```

Comment:
im not sure if you can trigger this on server side. dont trigger server functions like this. security issues